inclusive-design/idrc
# feat: remove Tailwind, improve previews

## Description

This PR removes the Tailwind CSS dependency and improves admin previews.

## Steps to test

1. Visit the site, check all pages.

**Expected behavior:** Designs should be consistent with current dev branch (https://dev--idrc.netlify.app).

## Additional information

Not applicable.

## Related issues

Not applicable.
